internal/ignition/ignition.go

```diff
@@ -411,3 +442,49 @@ func writeIgnitionFile(path string, config *config_31_types.Config) error {
 
 	return nil
 }
+
+func setFileInIgnition(config *config_31_types.Config, filePath string, fileContents string, mode int) error {
+	rootUser := "root"
+	file := config_31_types.File{
+		Node: config_31_types.Node{
+			Path:      filePath,
+			Overwrite: nil,
+			Group:     config_31_types.NodeGroup{},
+			User:      config_31_types.NodeUser{Name: &rootUser},
+		},
+		FileEmbedded1: config_31_types.FileEmbedded1{
+			Append: []config_31_types.Resource{},
+			Contents: config_31_types.Resource{
+				Source: &fileContents,
+			},
+			Mode: &mode,
+		},
+	}
+	config.Storage.Files = append(config.Storage.Files, file)
+	return nil
```

Comment:
This doesn't look like it can ever return an error, can you remove the return value?